bears/c_languages/ArtisticStyleBear.py

```diff
@@ -0,0 +1,865 @@
+                        return 1;
+                    } else
+                        return 0;
+                }
+
+            For example: Pico style uses broken braces and run-in statements
+            with attached closing braces. The closing brace is attached to the
+            last line in the block. Switches are indented to allow a run-in to
+            the opening switch block. This style frequently is used with an
+            indent of 2 spaces. If set to ``pico``, prefer::
+
+                int Foo(bool isBar)
+                {   if (isBar)
+                    {   bar();
+                        return 1; }
+                    else
+                        return 0; }
+
+            For example: Lisp style uses attached opening and closing braces.
+            The closing brace is attached to the last line in the block.
+            If set to ``lisp``,
+            prefer::
+
+                int Foo(bool isBar) {
+                if (isBar) {
+                    bar()
+                    return 1; }
+                else
+                    return 0; }
+
+        :param use_spaces:
+            In the following examples, ``q`` space is indicated with a ``.``
+            (dot), a tab is indicated by a > (greater than).
+            For example: If ``None``, the default option of 4 spaces will be
+            used as below::
+
+                void Foo() {
+                ....if (isBar1
+                ............&& isBar2)
+                ........bar();
+                }
+
+            For example: If set to ``True``, spaces will be used for
+            indentation.
+            For example: If set to ``False``, tabs will be used for
+            indentation, and spaces for continuation line alignment as below::
+
+                void Foo() {
+                >   if (isBar1
+                >   ........&& isBar2)
+                >   >   bar();
+                }
+
+        :param indent_size:
+            Number of spaces per indentation level.
+            For example: If ``use_spaces`` is ``True`` and ``indent_size`` is
+            ``3``, prefer::
+
+                void Foo() {
+                ...if (isBar1
+                .........&& isBar2)
+                ......bar();
+                }
+
+        :param attach_braces_to_namespace:
+            Attach braces to a namespace statement. This is done
+            regardless of the brace style being used.
+            For example: If set to ``True``, prefer::
+
+                namespace FooName {
+                ...
+                }
+
+        :param attach_braces_to_class:
+            Attach braces to a class statement. This is done regardless of the
+            brace style being used.
+            For example: If set to ``True``, prefer::
+
+                class FooClass {
+                ...
+                };
+
+        :param attach_braces_to_inline:
+            Attach braces to class and struct inline function definitions. This
```

Comment:
can you enclose all the PL related, settings values, PL names within backsticks...(so that they are easily differentiated while reading the docs)